shared/js/background/devbuild-reloader.js

```diff
@@ -4,10 +4,10 @@
  */
 
 import browser from 'webextension-polyfill'
-const browserWrapper = require('./wrapper')
+import { createAlarm, getFromSessionStorage, setToSessionStorage } from './wrapper'
```

Comment:
Nit: We have a convention of importing this one as browserWrapper. Might as well do that here too?
```suggestion
import * as browserWrapper from './wrapper'
```

Reply:
I would suggest we stop that convention and start only importing the individual library functions we need. In the long run, this enables the bundler to tree shake as it knows which functions are being used.